readthedocs/search/parse_json.py

```diff
@@ -101,9 +104,71 @@ def parse_content(content):
 
     # removing the starting text of each
     content = content.split('\n')
-    if len(content) > 1:  # there were \n
+    if remove_first_line and len(content) > 1:
```

Comment:
Was this the issue we were having with not getting the full docstring?

Reply:
Nope.
This is done because section content contains the section title as its first line.
Section title is already parsed and we don't need same thing twice.